e2e/test/scenarios/dashboard/dashboard-card-resizing.cy.spec.js

```diff
@@ -215,8 +215,11 @@ describe("scenarios > dashboard card resizing", () => {
         cy.findByLabelText("Add questions").click();
       });
 
+      cy.intercept("POST", /\/api\/card\/(pivot\/)?.*\/query/).as("card-query");
+
       TEST_QUESTIONS.forEach(question => {
         cy.findByLabelText(question.name).click();
+        cy.wait("@card-query");
       });
```

Comment:
Hi Nick! How is this helping fix the flake? The flake you linked seems to be associated with saving the dashboard. The `saveDashboard()` helper expects the edit bar text to not exist, but because the dashboard has not fully loaded the saved version, it fails.

Reply:
Well, I had it in my head that the test was failing because we were adding cards very quickly, with each card added causing a request, then attempting to save the dashboard which was another request, and things were bogging down causing the flake. The change will make the test wait before each card has loaded before attempting to add the next as well as ensure all cards have loaded before trying to save, causing it not get so overwhelmed. I could be wrong though... After looking at the flake again, I'm not quite as convinced 😓

Reply:
Does it make any difference if we add all these questions via API instead? We can still assert on their default size.

Reply:
The default sizes are defined in the frontend. I'm guessing it was done this way for that reason.

Reply:
Btw `**` will match anything, including `/`

```suggestion
      cy.intercept("POST", "/api/card/**/query/").as("card-query");

      TEST_QUESTIONS.forEach(question => {
        cy.findByLabelText(question.name).click();
        cy.wait("@card-query");
      });
```

https://github.com/metabase/metabase/blob/master/e2e/support/commands/api/question.js#L109:L112

Reply:
>If there's no other option, you can increase the timeout for "You're editing this dashboard."
The issue here is that the save was failing. Or at least the app thought it was. According to deploy sentinel the request was fine, but the button had the context of "Save failed"

Reply:
I took another look at DeploySentinal, and it looks like maybe the test isn't as flakey as I thought. I'll remove this for now and we can revisit it later if needed